util/okapi-curl

```diff
@@ -3,26 +3,56 @@
 # Edit the `.okapi` file in your home directory to contain something like:
 #
 #	OKAPI_URL=https://east-okapi.folio-dev.indexdata.com
-#	OKAPI_TENANT=reshare_east
```

Comment:
I think here you meant to delete the TOKEN line, not the TENANT line — is that right?

Reply:
No.